components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/webextension/GeckoWebExtension.kt

```diff
@@ -218,6 +222,25 @@ class GeckoWebExtension(
         val geckoSession = (session as GeckoEngineSession).geckoSession
         return geckoSession.getWebExtensionActionDelegate(nativeExtension) != null
     }
+
+    /**
+     * See [WebExtension.getMetadata].
+     */
+    override fun getMetadata(): Metadata? {
+        return nativeExtension.metaData?.let {
+            Metadata(
+                name = it.name,
```

Comment:
I guess we can't show this info to the user as I will be not localized, we have to rely on the addon counterpart, right?

Reply:
Yes, exactly. We're going to use what we have in our `Addon` in the UI, but I wanted us to have access to all fields in case we need them.

Reply:
The string are localized based on the GeckoView language setting.

Reply:
👍